content/attacks/posts/2019-11-27-Upbit.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2019-11-27
+custodians: Upbit
+categories: "Custodian"
+title: "Upbit Loses $49 Million in Cyberattack"
+---
+
+## Summary
+
+On November 27, 2019, Upbit, one of South Korea's largest cryptocurrency exchanges, fell victim to a major cyberattack, resulting in [the theft of approximately $49 million](https://www.coindesk.com/markets/2019/11/27/upbit-is-the-seventh-major-crypto-exchange-hack-of-2019/#:~:text=Upbit%20is%20the%20latest%20hacking,for%20at%20least%20two%20weeks.) worth of Ethereum (342,000 ETH). The attack specifically targeted Upbit's hot wallet, from which the funds were transferred to an unknown wallet address. To safeguard the remaining assets, Upbit promptly [moved them to cold storage](https://www.coindesk.com/markets/2020/01/14/upbit-exchange-resumes-ether-services-months-after-49m-hack/). The exchange took immediate action, [replenishing the stolen funds with its own assets](https://upbit.com/service_center/notice?id=1085&__cf_chl_jschl_tk__=0a2128c5bc6e7ba8d9f99fff770676b8070e471b-1579005041-0-AUmR6pokVjxLCmFXWKC9Y1atFSTinmPh0P8Kop58d7j4yWp9xNfodR6vQihzOV4dIzfY3uOOT6Dx736M0yQfRvuxbk9pTugKkB7c4-ISSZ9lxyO_YPwSbyhLNMTtYbUIcK76wr2jst46ThuLr5m4fuBNWfhXSbf6nt-WRpTSiKbYOe0MXcjIB3lOar_v1jKImE25ubqU1M0sKtu__DLaGz8pCBL4oPYNSSBAAA88GGFhXoBI51UqyowBWDD43U8xgIgOWODNJXRQLJLJVBhGMF8iGHyzb6lzOShTvEEQky9B) and ensuring no impact on its customers.
+
+## Attackers
+
+The identity of the attackers involved in the Upbit hack remains unknown. However, the wallet address associated with the attackers [has been annotated as](https://explorer.bitquery.io/ru/ethereum/address/0xa09871aeadf4994ca12f5c0b6056bbd1d343c029): *Upbit Theft*, *Upbit Hacker 1*, *Upbit Hack*, *Heist*. The address is 0xa09871AEadF4994Ca12f5c0b6056BBd1d343c029.
```

Comment:
Shorten the sentence, leave only one annotation or remove it completely. Make an address as a link, like [0xa09871AEadF4994Ca12f5c0b6056BBd1d343c029](onchain link) on a new line
And I would suggest to use Etherscan links, since it's more appropriate and Bitquery loadds slower